### PR TITLE
Release v19

### PR DESCRIPTION
**Pull request by sideninja**
### Features:
- Project deploy with contract init arguments #173 
- Network status command #163 
- Global configuration #213 
- Dot env support #214 
- Command autocompletion #203 
- Status command #163 
- Configuration manipulation command #218 

### Improvements:
--- (reverted) Remove multiple keys in configuration #216 
- Typo fixes #215 
- Default network without configuration #205 
- Chain ID removed #204 
- Extended integration tests #223 
- Send signed transaction progress

### Bugfixes:
- Transaction proposal key index #220 
- Transaction boolean argument 
- JSON output bugs fixed

### Release Scheduled: 26.04.2021

### CODE REVIEW

File: docs/initialize-configuration.md [hunk 1]
```diff
@@ -34,26 +34,50 @@ or by removing the configuration file first.
 ❌ Command Error: configuration already exists at: flow.json, if you want to reset configuration use the reset flag
 ```
 
+## Global Configuration
+
+Flow supports global configuration which is a `.flow.json` file saved in your home
```
Comment (psiemens) [inline docs/initialize-configuration.md:39]: Does it need to be a dotfile? I worry that users will become confused that the global version is `.flow.json` wheres the local version is `flow.json`.

Comment (sideninja) [reply docs/initialize-configuration.md]: Done

File: pkg/flowcli/config/config.go [hunk 2]
```diff
@@ -157,6 +168,26 @@ func DefaultEmulators() Emulators {
...
+	dirname, err := os.UserHomeDir()
+	if err != nil {
+		return ""
+	}
+
+	return fmt.Sprintf("%s/%s", dirname, DefaultPath)
+}
+
+// DefaultPaths determines default paths for configuration
+func DefaultPaths() []string {
```
Comment (psiemens) [inline pkg/flowcli/config/config.go:184]: Nice 👍

**Review by psiemens — APPROVED** [review]

File: internal/accounts/accounts.go [hunk 3]
```diff
@@ -32,6 +31,7 @@ var Cmd = &cobra.Command{
 	Use:              "accounts",
 	Short:            "Utilities to manage accounts",
 	TraverseChildren: true,
+	ValidArgs:        []string{"get", "contract-add"},
```
[the remark this answers is not in the document]
Comment (sideninja) [reply internal/accounts/accounts.go:34]: nope, already deleted, not sure how that stayed there, I was doing some testing for command autocompletion

**Pull request merged**